docs/rfcs/XeTile.md

```diff
@@ -25,60 +25,109 @@ XeTile provides a middle-level abstraction for matmul operation and sits between
+
+```mlir
+  #tile_attr = #xetile.tile_ttr<order = [0, 1]>
+  %tile0 = XeTile.init_tile %base_memref, [%tile_offset:2]:
+     memref<128x128xbf16> into tile<64x32xbf16, #tile_attr>
+```
+
+`init_tile` with an `inner_block` for 2D block access of the base matrix. The `inner_blocks` attribute describes the block size for each memory load and store operation when the tile is being loaded. The block size for load may be larger than the block size for MMA operation. The output tile carries the `inner_block` attribute in its attribute set. 
+
+```mlir
+  #tile_attr = #xetile.tile_ttr<inner_blocks=[16,16]>
+  %tile0 = XeTile.init_tile %base_memref, [%tile_offset:2]:
+     memref<128x128xbf16> into tile<64x32xbf16, #tile_attr>
+```
+
 With the tile date type, XeTile supports load_tile, prefetch_tile, and store_tile.
 
-`load_tile` loads a tile to a vector, which could be backed by a register region.
+`load_tile` loads a tile to a 2D vector, which could be backed by a register region.
 ```mlir
   %vector_a = XeTile.load_tile %tile_a :
      tile<64x64xbf16> into vector<64x64xb16>
 ```
-Attribute `transpose` specifies the dimensions being transposed along the load. It is commonly used for the GEMM on the backward path of DNN model, where one of input matrices needs to be transposed for matmul operation.
-```mlir
-  %vector_a = XeTile.load_tile %tile_a {transpose = [1, 0]} :
-     tile<32x64xbf16> into vector<64x32xbf16>
-```
+
 Attribute `padding` specifies the padding value for the out-of-boundary access. The default value is zero.  
 ```mlir
   %vector_a = XeTile.load_tile %tile_a {padding = 1.0} :
      tile<64x64xbf16> into vector<64x64xb16>
 ```
-`load_tile` need to be used together with the tile_mma.
+`load_tile` needs to be used together with the tile_mma.
+
+`load_tile` loads a tile with an `order` attribute. The `order` attribute only affects the physical memory address calculation for the tile, however, it doesn't change the logical representation of loading a 2D tile to a 2D vector. There is no need to reverse the order of vector length at the XeTile level regardless of the `order` attribute value. 
+```mlir
+  #tile_attr = #xetile.tile_ttr<order = [0, 1]>
+  %vector_a = XeTile.load_tile %tile_a :
+     tile<64x32xbf16> into vector<64x32xb16, #tile_attr>
+```
+
+`load_tile` loads a 2D tile with an `inner_block` attribute  to 4D vector.
+```mlir
+  #tile_attr = #xetile.tile_ttr<inner_blocks=[16,16]>
+  %vector_a = XeTile.load_tile %tile_a :
+     tile<64x32xbf16> into vector<4x2x16x16xb16, #tile_attr>
+```
 
 `store_tile` stores a vector to memory. Transpose and padding attributes are not supported.
 ```mlir  
   XeTile.store_tile %tile_a, %vector_a :
    vector<64x64xbf16> into tile<64x64xbf16>
 ```
+`store_tile` loads a tile with an `order` attribute. With the `order` attribute, there is no need to reorder the 2D vector dimension before saving it to a column-major matrix storage. 
```

Comment:
"loads a tile" => "stores a tile"
"with an" =>  "according to the"
"dimension before saving it to a column-major matrix storage."
"dimensions. The op does the reordering."